pkg/client/client.go

```diff
@@ -0,0 +1,29 @@
+package client
+
+import (
+	"github.com/bitnami-labs/charts-syncer/pkg/client/types"
+	"helm.sh/helm/v3/pkg/chart"
+)
+
+// Reader defines the methods that a ReadOnly chart client should implement.
```

Comment:
> chart client

This comment might be now outdated, it might make sense to add some more literature to this package documentation

Reply:
Added something, but not really sure if that's enough